src/ray/protobuf/gcs_service.proto

```diff
@@ -624,6 +624,8 @@ message GetTaskEventsRequest {
   // If set, the exact `limit` TaskEvents returned do not have any ordering or selection
   // guarantee.
   optional int64 limit = 3;
+  // True if only task events with non-empty task state changes should be returned.
```

Comment:
Hmm this flag seems a bit confusing to me. Maybe it should be exclude_driver_task?

Reply:
I am ok with having `driver` mentioned in the flag - although it's a bit implicit where a driver task will generate task events w/o any status updates but only profiling events.